applications/serial_lte_modem/src/slm_at_tcp_proxy.c

```diff
@@ -8,10 +8,13 @@
 #include <stdio.h>
 #include <string.h>
 #include <zephyr/net/socket.h>
+#include <zephyr/net/tls_credentials.h>
```

Comment:
```suggestion
```

Reply:
Needed for sec_tag_t when no native tls.